packages/@scope/jsii-calc-lib/test/assembly.jsii

```diff
@@ -179,7 +182,7 @@
       "docs": {
         "remarks": "These classes can be greeted with\na \"hello\" or \"goodbye\" blessing and they will respond back in a fun and friendly manner.",
         "stability": "deprecated",
-        "summary": "Applies to classes that are considered friendly."
+        "summary": "Applies to classes that are considered friendly. (deprecated)"
```

Comment:
I think deprecation should be a prefix, not a postfix:

    (DEPRECATED) Applies to classes to are considered friendly

Also, where is the deprecation message going?